src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1491,6 +1491,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return typesXmlConverter;
```

Comment:
@iSazonov, your last commit had 65 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

ConsoleHost unit tests.Clear-Host does not injects data into PowerShell output stream
```powershell
Multiple ambiguous overloads found for "SetBufferContents" and the argument count: "2".
at <ScriptBlock>, : line 4
4: $RawUI.SetBufferContents(

```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable -All gets all compatible modules when SkipEditionCheck: $true, using root module: $true, using absolute path: $false, CompatiblePSEditions:
```powershell
Cannot convert argument "value", with value: "f3d5493c-9579-4f13-9373-f0fc355c3026", for "Contains" to type "System.Char": "Cannot convert the "f3d5493c-9579-4f13-9373-f0fc355c3026" value of type "System.Guid" to type "System.Char"."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 534
534:                 Get-Module -ListAvailable -All -SkipEditionCheck | Where-Object { $_.Path.Contains($guid) }

```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable -All gets all compatible modules when SkipEditionCheck: $true, using root module: $false, using absolute path: $false, CompatiblePSEditions
```powershell
Cannot convert argument "value", with value: "f077c8e4-f271-4789-b00b-257ea5179535", for "Contains" to type "System.Char": "Cannot convert the "f077c8e4-f271-4789-b00b-257ea5179535" value of type "System.Guid" to type "System.Char"."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 534
534:                 Get-Module -ListAvailable -All -SkipEditionCheck | Where-Object { $_.Path.Contains($guid) }

```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable -All gets all compatible modules when SkipEditionCheck: $false, using root module: $true, using absolute path: $false, CompatiblePSEditions
```powershell
Cannot convert argument "value", with value: "6207bcf0-b521-41ca-9563-322aa053778d", for "Contains" to type "System.Char": "Cannot convert the "6207bcf0-b521-41ca-9563-322aa053778d" value of type "System.Guid" to type "System.Char"."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 538
538:                 Get-Module -ListAvailable -All | Where-Object { $_.Path.Contains($guid) }

```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable -All gets all compatible modules when SkipEditionCheck: $false, using root module: $false, using absolute path: $false, CompatiblePSEdition
```powershell
Cannot convert argument "value", with value: "aeace227-ebdd-4257-9746-7df11c16e938", for "Contains" to type "System.Char": "Cannot convert the "aeace227-ebdd-4257-9746-7df11c16e938" value of type "System.Guid" to type "System.Char"."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 538
538:                 Get-Module -ListAvailable -All | Where-Object { $_.Path.Contains($guid) }

```